google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -162,12 +180,22 @@ public void addGet(StorageObject storageObject, RpcBatch.Callback<StorageObject>
 
     @Override
     public void submit() {
```

Comment:
I didn't instrument `addGet`, `addDelete`, `addPatch` since I think what they do is just add something into the queue. Please correct me if I am wrong.

Reply:
I believe you're correct.